### DISCUSSION TITLE
How to add a small brandLogo when sidebar is collapsed ?

### DISCUSSION THREAD

**Discussion by pranab-acharya** [filamentphp/filament#12801] [Help]
Unable to add brandLogo when sidebar is collapsed, any help will be appreciated.

@danharrin

![image_2024_05_15T10_06_06_777Z](https://github.com/filamentphp/filamentphp.com/assets/152287594/ddc9ac6e-9f0f-499e-88a9-78219e1de146)

**Comment by leandrocfe** [comment]
Creating a custom theme and customizing it?
https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme

**Reply by pranab-acharya** [reply]
Yes yes obviously, we can do that by creating our new theme, but i am asking if there is any option in filament by default to achieve this.
